src/test/resources/internationalization/i18NCases_en.properties

```diff
@@ -21,4 +21,4 @@ popup.message.regexp=Case "[0-9]{8}" was created.
 title.case=Case
 header.caseNumber=@title='Case Number'
```

Comment:
why is title in the properties? does it change in Spanish too?

Reply:
It is a similar issue as the previous comment. On english there is no problem but on spanish it has accent. To make the locator more general I sent the entire value. This is how it looks on the spanish version:
starts-with(@title,'N') and contains(@title,'mero del caso').
As the other, it would demand some refactor.